support-frontend/assets/components/legal/termsPrivacy/termsPrivacy.scss

```diff
@@ -10,9 +10,8 @@
   }
 }
 
-.patrons, .philanthropic-ask {
+.patrons, .philanthropic-ask{
```

Comment:
tiny point - I think we still want this space?

Reply:
yes, good spot! I'll add it back in